microsetta_admin/server.py

```diff
@@ -774,6 +775,191 @@ def scan():
                                       recorded_type)
 
 
+def get_legends(criteria):
```

Comment:
Functions that are only used internally should begin with an underscore (e.g. `_get_legends()`)

Reply:
Resolved